modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/upload/BaseMBUploadFileEntryHandler.java

```diff
@@ -58,10 +63,7 @@ public FileEntry upload(UploadPortletRequest uploadPortletRequest)
 
 	protected abstract String getParameterName();
 
-	@Reference
 	protected DLValidator dlValidator;
```

Comment:
Hey @regiCesar21, these fields should be `private final` since they are just used internally. Thx